ROOT_PATH: /home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language
Building Bayesian Optimizer for 
 data:IMDB 
 choices:[{'domain': [5e-05, 0.005], 'name': 'lr', 'type': 'continuous'}, {'domain': [0, 1], 'name': 'rnn_dropout', 'type': 'continuous'}, {'domain': [0, 1], 'name': 'dropout', 'type': 'continuous'}, {'domain': [0, 1], 'name': 'l2', 'type': 'continuous'}]
SETTINGS FOR THIS RUN
{'dropout': 0.29395660827394643, 'tune_wordvecs': True, 'wordvec_dim': 300, 'batch_size': 32, 'wordvec_source': 'google', 'num_layers': 3, 'rnn_dropout': 0.7250988849440482, 'attn_type': None, 'lr': 0.002837798012759877, 'l2': 0.6426628785345077, 'data': 'IMDB'}
Using CUDA!
here
None
Building RNN Classifier...

[93m    Warning: no model found for 'en'[0m

    Only loading the 'en' tokenizer.

Retrieving Data from file: /home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language/data/imdb/aclImdb/train...
Got 25000 examples in 63.8459358215332 seconds
Retrieving Data from file: /home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language/data/imdb/aclImdb/test...
Got 25000 examples in 61.19954586029053 seconds
Loading Vectors From Memory...
Using these vectors: ['googlenews']
Getting google news vectors
Building Vocab...
[<torchtext.vocab.Vectors object at 0x7fd559173ac8>]
Getting Batches...
Created Iterator with 782 batches
Getting Batches...
Created Iterator with 782 batches
Building model...
Traceback (most recent call last):
  File "tune_models.py", line 186, in <module>
    batch_size = args.batch_size, seq_len = args.seq_len)
  File "tune_models.py", line 67, in __init__
    exact_feval = True
  File "/home-nfs/siddsach/anaconda3/lib/python3.5/site-packages/GPyOpt/methods/bayesian_optimization.py", line 117, in __init__
    self._init_design_chooser()
  File "/home-nfs/siddsach/anaconda3/lib/python3.5/site-packages/GPyOpt/methods/bayesian_optimization.py", line 192, in _init_design_chooser
    self.Y, _ = self.objective.evaluate(self.X)
  File "/home-nfs/siddsach/anaconda3/lib/python3.5/site-packages/GPyOpt/core/task/objective.py", line 50, in evaluate
    f_evals, cost_evals = self._eval_func(x)
  File "/home-nfs/siddsach/anaconda3/lib/python3.5/site-packages/GPyOpt/core/task/objective.py", line 74, in _eval_func
    rlt = self.func(np.atleast_2d(x[i]))
  File "tune_models.py", line 119, in getError
    optimizer = trainer.start_train()
  File "/home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language/classifier/attention_rnn/trainer.py", line 588, in start_train
    self.get_model(pretrained_weights, pretrained_args)
  File "/home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language/classifier/attention_rnn/trainer.py", line 387, in get_model
    self.model = VanillaRNN(**args)
  File "/home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language/classifier/attention_rnn/model.py", line 70, in __init__
    self.init_embedding(vectors)
  File "/home-nfs/siddsach/ml/Interpreting-Attention/interpreting_language/classifier/attention_rnn/model.py", line 73, in init_embedding
    self.embed.weight.data.copy_(pretrained_embeddings)# this provides the values
RuntimeError: inconsistent tensor size, expected tensor [101925 x 300] and src [101925 x 100] to have the same number of elements, but got 30577500 and 10192500 elements respectively at /opt/conda/conda-bld/pytorch_1513366702650/work/torch/lib/TH/generic/THTensorCopy.c:86
